common/src/main/java/com/microsoft/identity/common/sharedwithoneauth/OneAuthSharedFunctions.kt

```diff
@@ -45,6 +46,7 @@ class OneAuthSharedFunctions {
          * @param activeBrokerPackageName name of the app hosting the broker process to communicate to.
          **/
         @JvmStatic
+        @WorkerThread
```

Comment:
This is a breaking change for oneauth

Reply:
I can remove this. Meant to be a reminder only (to make sure we're slowly moving this away from main thread)